src/lib/Behat/PageObject/UserCreationPage.php

```diff
@@ -50,12 +50,19 @@ public function verifyTitle(): void
     public function verifyRoute(): void
     {
         $expectedPageRoutes = ['/user/create/user', '/user/update'];
-        $actualRoute = $this->getPageRoute();
+        $actualRoute = $this->getCurrentRoute();
         foreach ($expectedPageRoutes as $expectedPageRoute) {
             if (strpos($actualRoute, $expectedPageRoute) !== false) {
                 return;
             }
         }
-        Assert::fail(sprintf('Expected: %s Actual: %s', '/user/update', $actualRoute));
+        Assert::fail(sprintf('Expected: %s Actual: %s', $expectedPageRoutes, $actualRoute));
```

Comment:
This will fail with a PHP error - $expectedPageRoutes is an array, you can't cast it to string without any issues.

You can use:
```
Assert::fail(sprintf('Expected one of: %s. Actual: %s', implode(',', $expectedPageRoutes), $actualRoute));
```